src/config/constants.js

```diff
@@ -70,3 +70,22 @@ export const SUPPORTED_NETWORKS = {
     identityServiceAppId: process.env.NODE_ENV === 'test' ? 'appid' : ROPSTEN_IDENTITY_SERVICE_APPID,
   },
 };
+
+if (process.env.MINI_CLUSTER_HOST) {
```

Comment:
We can run this version on top of the minicluster? cooool!

Reply:
Yes, it supports the local DEV environment with mini cluster. 😄